Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM golang:1.14-stretch
-RUN apt-get update && apt-get install git 
-
-RUN go get github.com/richardlehane/siegfried/cmd/sf && /go/bin/sf -update
-CMD cp /root/siegfried/default.sig /tmp/fileformatbuild && cp /go/bin/sf /tmp/fileformatbuild
+FROM openjdk:8-jdk
```

Comment:
Out of curisity, do you know if we can use a more up-to-date JDK? I haven't been paying enough attention to what JDK the Scala code has been using.

Reply:
droid 6.5 is compatible with openjdk11

Reply:
So the way this originally worked was that I used the jdk8 image as the base and copied the jre folder onto the EFS volume and used that to run droid. I stopped doing that though because the file-format lambda uses the Java runtime so it's pointless.
I changed the runtime to java11 in the [terraform-modules](https://github.com/nationalarchives/tdr-terraform-modules/pull/45/commits/4d18430656ede4d08a26ad43ae1cca27bd83e2bf) and it's working.
I've also changed the Dockerfile to use alpine as the base image as all it's doing now is wgetting droid and unzipping it. This has reduced the image size from ~600Mb to about 40Mb which is nice.